convert2rhel/systeminfo.py

```diff
@@ -154,6 +154,27 @@ def _generate_rpm_va(self):
         self.logger.info("The 'rpm -Va' output has been stored in the %s file"
                          % output_file)
 
+    def generate_modified_rpms(self):
+        """Generate a log and stdout message with modified rpms after the convert process """
+        if tool_opts.no_rpm_va:
+            self.logger.info("Skipping execution of 'rpm -Va'.")
+            return
+
+        before_rpm_va = utils.get_file_content(os.path.join(logger.LOG_DIR, "rpm_va.log"), True)
+
+        rpm_va, _ = utils.run_subprocess("rpm -Va", print_output=False)
+        output_file = os.path.join(logger.LOG_DIR, "rpm_va_after_conversion.log")
+        utils.store_content_to_file(output_file, rpm_va)
+        difference = list(set(before_rpm_va).symmetric_difference(set(rpm_va.split())))
```

Comment:
Can be refactored to reduce code duplication with

https://github.com/oamg/convert2rhel/blob/8a31c2c17eac2628a2d29e1fc1b5a717d180a6e7/convert2rhel/systeminfo.py#L140-L154